components/org.wso2.carbon.identity.oauth/src/main/java/org/wso2/carbon/identity/oauth2/token/OauthTokenIssuerImpl.java

```diff
@@ -34,6 +43,23 @@ public class OauthTokenIssuerImpl implements OauthTokenIssuer {
     private boolean persistAccessTokenAlias = true;
 
     public String accessToken(OAuthTokenReqMessageContext tokReqMsgCtx) throws OAuthSystemException {
+        String renewWithoutRevokeConfig =
+                IdentityUtil.getProperty(RENEW_TOKEN_WITHOUT_REVOKING_EXISTING_ENABLE_CONFIG_FOR_OPAQUE);
+
+        boolean renewWithoutRevokingExistingEnabled = false;
+        if (renewWithoutRevokeConfig != null) {
```

Comment:
Boolean.parseBoolean() will return false if the value is null. We can use that directly here without this if condition